.circleci/config.yml

```diff
@@ -73,6 +76,10 @@ workflows:
             parameters:
               ruby_version:
                 - 2.6.6
+                - 2.7.1
+                - 2.7.2
                 - 2.7.3
+                - 2.7.4
+                - 2.7.5
```

Comment:
While it is still not clear what the highest available version will be from Brightbox, I figured we may as well test against 'em all. It's probably overkill, but it doesn't slow anything down, and will provide a high degree of confidence when we're ready to switch versions in production. We can remove the unneeded 2.7 versions when we remove 2.6.6.